src/Vlingo.Actors/World.cs

```diff
@@ -425,7 +425,14 @@ public void RegisterDynamic(string name, object dep)
         /// <returns></returns>
         public TDependency ResolveDynamic<TDependency>(string name)
         {
-            return (TDependency)this.dynamicDependencies[name];
+            try
```

Comment:
Sorry @jamesmh. I didn't check properly as I was on mobile but it would be much more efficient to use `bool TryGetValue (TKey key, out TValue value);` instead of `try/catch` block. Could you please make this late change.

Reply:
NP. I should have known better 🙃